### DISCUSSION TITLE
Custom Site-Width

### DISCUSSION THREAD

**Discussion by tvories** [Lednerb/bilberry-hugo-theme#446] [Q&A] [answered]
It's been a while since https://github.com/Lednerb/bilberry-hugo-theme/issues/155 was opened and I wanted to see if it were possible to provide an override to set the site width to something larger than 800px.  I know I can override it with the SASS, but is something like this possible to set up without having to customize the theme?

**Comment by igor-baiborodine** [comment, marked as answer]
@tvories, to my knowledge, it's not possible. @Lednerb, please correct me if I'm wrong.

**Reply by Lednerb** [reply]
It would be possible but would also need a lot of refactoring.

There once was on option that used the hugo pipelines to dynamically build the scss files to css on the `hugo` command.
Later I've removed it because the usage was not easy and it needed a lot of maintenance.

It can be done by either using relative sizes (`rem` and `em` instead of `px`) or calculating some functions for spaces, margins, paddings etc. so it would also work on small / mobile screens and big ones (oversized curved monitos / TVs).

IMHO it's not worth the extra amount of time needed for implementation.

**Reply by tvories** [reply]
Thanks for the info, and I was unaware of the limitations.  It isn't a huge deal, it just feels like a bit of wasted space with a larger screen.
